### ISSUE TITLE
Research an Installer

### ISSUE THREAD

**Issue by kellyelton** [octgn/OCTGN4#3]
This is mostly because I ran across this again after 3 years and apparently it's actually working and awesome.

[https://github.com/Squirrel/Squirrel.Windows](https://github.com/Squirrel/Squirrel.Windows)

Either way we'll need an installer, so that's what this is for.

**Comment by Gravecorp** [comment]
It looks good. have to research if we can automate the build stuff on our build server to do this. its basically a nuget wrapper with an installer and update mechanism. so that is good. does add quite a bit of libs to make it work though.

**Comment by kellyelton** [comment]
Yeah. I didn't do a lot of research on it lately, but was talking to someone who uses it and apparently it's really nice now. Honestly I don't know.

Some other ones I think we could investigate are
- Wix
- Nullsoft

Nice thing I think about picking something newer is I think it might not feel as hacky as nullsoft.
